packages/node-sdk/example/bucket.ts

```diff
@@ -0,0 +1,13 @@
+import { BucketClient } from "@bucketco/node-sdk/src";
+
+export const bucket = new BucketClient({
+  secretKey: process.env.BUCKET_SECRET_KEY || "sec_prod_12345678901234567890",
```

Comment:
Let's remove the fallback? Maybe too late, but maybe it'll remove some false security warnings from scanners. I've added a suggestion to the README instead.

Reply:
It just was a fake key, I was testing

Reply:
yes of course, but the security scanners don't know that :)